projects/step-frontend/src/lib/modules/timeseries/execution-page/execution-page.component.ts

```diff
@@ -70,6 +92,30 @@ export class ExecutionPageComponent implements OnInit, OnDestroy {
     });
   }
 
+  rebuildTimeSeries() {
+    this.migrationInProgress = true;
+    this.timeSeriesService
+      .rebuildTimeSeries(this.executionId)
+      .pipe(
+        pollAsyncTask(this._asyncTaskService),
+        delay(10_000) // in order for the migration to be completed as much as possible
```

Comment:
I'm confused about the delay, the pollAsyncTask should have its internal 500ms delay and only return if the task is ready.

Reply:
Agree. delay is redunant here